test/js/bun/dns/resolve-dns.test.ts

```diff
@@ -6,7 +6,7 @@ import { isIP, isIPv4, isIPv6 } from "node:net";
 const backends = ["system", "libc", "c-ares"];
 const validHostnames = ["localhost", "example.com"];
 const invalidHostnames = ["adsfa.asdfasdf.asdf.com"]; // known invalid
-const malformedHostnames = ["", " ", ".", " .", "localhost:80", "this is not a hostname"];
+const malformedHostnames = [" ", ".", " .", "localhost:80", "this is not a hostname"];
```

Comment:
Why is the empty case removed?

Reply:
If this is because of the new case for `ENODATA`, I don't see a test for it.

Reply:
It returns a different error on empty string, and previously `malformedHostnames` was not used at all